node/service/src/rpc.rs

```diff
@@ -188,11 +185,19 @@ where
 		eth_log_block_cache,
 	));
 
+	struct FakeTransactionConverter;
```

Comment:
Wondering, is it possible to further restrict this arg type in Frontier to an `Option<Box<dyn ConvertTransaction>>` so here we can just set it to `None`? Having to set a fake implementor to just satisfy the rpc handler end seems unnecessary.

Reply:
@tgmichel I finally used a non-instantiable type (an enum without variant), to have the guarantee at compile time that the transaction_converter can't be called.